components/molecules/Quote.js

```diff
@@ -0,0 +1,46 @@
+import PropTypes from "prop-types";
+
+/**
+ * Quote component
+ */
+export function Quote(props) {
+  return (
+    <div
+      title="Quote"
+      className="border-l-4 text-sm md:text-p pl-4 leading-normal font-body"
+    >
+      {props.html === undefined ? (
+        <div>{props.children}</div>
+      ) : (
+        <div dangerouslySetInnerHTML={{ __html: props.html }} />
```

Comment:
Qoute component should only take text. There isn't really a need for dynamic content. It's just text and an author